### DISCUSSION TITLE
Evaluating Medley compatibility with Practical Common Lisp code: Chapter 15

### DISCUSSION THREAD

**Discussion by pamoroso** [Interlisp/medley#1593] [General]
## 15. Practical: A Portable Pathname Library

This is a continuation of the work on issue #609. To evaluate the compatibility of Medley Common Lisp with ANSI Common Lisp I'm testing on Medley the code of the book [Practical Common Lisp](https://gigamonkeys.com/book) by Peter Seibel.

[Chapter 15](https://gigamonkeys.com/book/practical-a-portable-pathname-library) describes the implementation of a portable pathname library. Since the code is written for and relies on specific features of modern Common Lisp systems, a complete reimplementation or a major adapation would be necessary for running it on Medley. So for the time being I'm skipping this chapter.

**Comment by masinter** [comment]
What does PCL say about packages that Medley doesn't implement?
a PATHNAME issue would be useful.

**Reply by pamoroso** [reply]
So far I completed chapter 15 of the book and packages are covered in chapter 21, so I don't know yet whether there are any major package issues.

I opened issue #1598 for reporting the missing pathnames features.
